docs/plugins/backend-plugin.md

```diff
@@ -116,36 +97,172 @@ curl localhost:7007/api/carmen/health
 
 This should return `{"status":"ok"}` like before. Success!
 
+## Secure by Default
+
+In 1.25, Backstage started moving to a secure by default model for plugins. This means that network requests to plugins will by default not allow unauthenticated users. Let's take a deeper look at the above curl request which should allow unauthenticated access.
+
+The actual endpoint that is being called is defined in
+
+```ts title="plugins/carmen-backend/src/service/router.ts"
+export async function createRouter(
+  options: RouterOptions,
+): Promise<express.Router> {
+  const { logger } = options;
+
+  const router = Router();
+  router.use(express.json());
+
+  // highlight-start
+  router.get('/health', (_, response) => {
+    logger.info('PONG!');
+    response.json({ status: 'ok' });
+  });
```

Comment:
can this example and the one below be cut down to only the effective statements that are being pointed out? like the entire `router.get` and the `httpRouter.addPolicy` statements. There's so much boilerplate around them that you have to dig through as a reader, and that we have to maintain over time.

Reply:
Yup, I'll adjust tonight

Reply:
updated in [453f22d](https://github.com/backstage/backstage/pull/24069/commits/453f22dd4f452c86766b02e22af6fa22f107a4d3)